### PR TITLE
Add support for sidecar container in statefulset

### PR DESCRIPTION
**Pull request by aslubsky**
#### What this PR does / why we need it:
Add support for sidecar container in statefulset. Weneed to run additinal container in zammad pod for backup storage file into aws s3.

#### Checklist

- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: zammad/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: zammad
-version: 6.4.1
+version: 6.4.2
```
Comment (monotek) [inline zammad/Chart.yaml:3]: new feature should raise minor version.
```suggestion
version: 6.5.0
```

**Review by monotek — CHANGES_REQUESTED** [review]

**Review by monotek — CHANGES_REQUESTED** [review]
can we please also add an disabled example to the values file?

**Comment by aslubsky** [comment]
> can we please also add an disabled example to the values file?

done

**Review by monotek — CHANGES_REQUESTED** [review]
please move above: https://github.com/zammad/zammad-helm/blob/43c3d9132add487115e4ce185944d1969038d0a0/zammad/values.yaml#L306

**Comment by aslubsky** [comment]
> please move above:
> 
> https://github.com/zammad/zammad-helm/blob/43c3d9132add487115e4ce185944d1969038d0a0/zammad/values.yaml#L306

done

**Review by monotek — APPROVED** [review]

**Pull request merged**